### PR TITLE
config_sync

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 ---
 skip-changelog:
- - head-branch: ['^release-*', 'release']
+ - head-branch: ['^release-*']
```
Comment (bastelfreak) [inline .github/labeler.yml:3]: why did you remove the `release` branch?

File: .github/release.yml [hunk 2]
```diff
@@ -4,7 +4,14 @@
 changelog:
   exclude:
     labels:
+      - duplicate
+      - invalid
+      - modulesync
```
Comment (bastelfreak) [inline .github/release.yml:9]: I don't think we use the modulesync label in this project?

File: .github/labeler.yml (hunk 1 above)
Comment (rwaffen) [reply .github/labeler.yml:3]: This is a pattern. Not a name. And if somehow Release ist in the name. The label would be applied

File: .github/release.yml (hunk 2 above)
Comment (rwaffen) [reply .github/release.yml:9]: Copy and past error 😅

**Comment by rwaffen** [comment]
closing here. this will be done with modulesync in another PR
